content/docs/faq-ajax.md

```diff
@@ -82,3 +82,47 @@ class MyComponent extends React.Component {
   }
 }
 ```
+
+Has the following equivalent with [Hooks](https://reactjs.org/docs/hooks-intro.html):
+
+```js
+function MyComponent() {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [items, setItems] = useState([]);
+
```

Comment:
Perhaps we should include a comment here to remind readers the empty deps array `[]` means this `useEffect` will run once similar to `componentDidMount()`?

Reply:
Done